config/kubernetes/sig-docs/teams.yaml

```diff
@@ -160,6 +171,8 @@ teams:
     - tnir # Japanese
     - zacharysarah # English
     - zhangxiaoyu-zidif # Chinese
+    - femrtnz # Portuguese
```

Comment:
@femrtnz Please keep this list alphabetical.

Reply:
good catch, thanks!